stdlib/gc.ml

```diff
@@ -61,9 +61,9 @@ external major : unit -> unit = "caml_gc_major"
 external full_major : unit -> unit = "caml_gc_full_major"
 external compact : unit -> unit = "caml_gc_compaction"
 external get_minor_free : unit -> int = "caml_get_minor_free"
-external get_bucket : int -> int = "caml_get_major_bucket" [@@noalloc]
-external get_credit : unit -> int = "caml_get_major_credit" [@@noalloc]
-external huge_fallback_count : unit -> int = "caml_gc_huge_fallback_count"
+(* external get_bucket : int -> int = "caml_get_major_bucket" [@@noalloc] *)
+(* external get_credit : unit -> int = "caml_get_major_credit" [@@noalloc] *)
+(* external huge_fallback_count : unit -> int = "caml_gc_huge_fallback_count" *)
```

Comment:
Why are those functions still present but commented -out?

Reply:
I think it's because there's an open question as to whether we remove them or provide implementations that might differ significantly from what existing callers expect.

Probably worth us removing these for now and creating an issue to re-examine.

Reply:
Removed in 5aa07b2